src/Libraries/RevitNodes/RevitNodes.csproj

```diff
@@ -57,11 +57,7 @@
     <Reference Include="ProtoGeometry">
       <HintPath>$(DYNAMOAPI)\ProtoGeometry.dll</HintPath>
       <Private>False</Private>
-    </Reference>
-    <Reference Include="RevitAPI">
-      <HintPath>$(REVITAPI)\RevitAPI.dll</HintPath>
-      <Private>False</Private>
-    </Reference>
+    </Reference>    
     <Reference Include="System" />
```

Comment:
why we change this project?

Reply:
There is two RevitAPI.dll reference in the same csproj, I think it could be delete one.

Reply:
Sorry where is the other reference you mentioned? Which line in this file?

Reply:
@QilongTang line 49-52 is the same as the line 61-64, this is repeated.